src/main/java/Entities/Player.java

```diff
@@ -143,33 +209,64 @@ public String rollDice() {
         return (a + "\n" + b);
     }
 
+    /**
+     * Builds a house on the current property
```

Comment:
We'll probably change the javadoc for this method as well since we are taking into consideration conditions that addHouses main fail and we probably need to return a boolean that indicates if addHouses is successful or not.